src/manifestManager.ts

```diff
@@ -184,8 +184,14 @@ export class ManifestManager implements vscode.Disposable {
      * @returns the selected manifest
      */
     async selectManifest(): Promise<Manifest | null> {
-        const quickPickItems: ManifestQuickPickItem[] = []
         const manifests = await this.getManifests()
+
+        if (manifests.isEmpty()) {
```

Comment:
I wonder if it would be too noisy when you open whatever project and you get that notification because it is a library or some random thing

Reply:
Maybe making use of the status bar is a better option here

Reply:
The notification won't show unless the user runs the `select-manifest` command

Reply:
I think this should also be the time where we rename manifest to Flatpak manifest, as it may be mistaken with other manifests

Reply:
In this specific case indeed, being more precise is better